### PR TITLE
A variety of whitespace fixes

### PR DESCRIPTION
**Pull request by sarahsturgeon**

### CODE REVIEW

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
retest this please

File: lua/cfcfactions/core/cl/cl_clientstartup.lua [hunk 1]
```diff
@@ -6,26 +6,26 @@ cfcFactions.MainDerma = nil
...
+-- Registers items to be placed into the menubar at loadtime
 function cfcFactions:RegisterDermaMenu( name, panel, ranking )
     if not CLIENT then return end
 
     if name == nil then name = "No Text Set" end
     if panel == nil then panel = {} end
     if ranking == nil then ranking = 99 end
     table.insert( cfcFactions.Dermas, {
-        internal_name=name, internal_panel=panel, internal_ranking=ranking, internal_button={}
+        internal_name = name, internal_panel = panel, internal_ranking = ranking, internal_button = {}
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/cl/cl_clientstartup.lua]:
```suggestion
table.insert( cfcFactions.Dermas, {
        internal_name = name,
        internal_panel = panel,
        internal_ranking = ranking,
        internal_button = {}
    }
)
```

File: lua/cfcfactions/core/cl/cl_clientstartup.lua [hunk 2]
```diff
@@ -6,26 +6,26 @@ cfcFactions.MainDerma = nil
...
     } )
 end
 
 function cfcFactions:AddAlert( msg, mtype )
     if mtype == nil then mtype = MsgType.Msg end
-    table.insert( cfcFactions.Alerts, {["Message"]=msg, ["Type"]=mtype, ["Time"]=os.date( "%T ", os.time() )} )
+    table.insert( cfcFactions.Alerts, {["Message"] = msg, ["Type"] = mtype, ["Time"] = os.date( "%T ", os.time() )} )
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/cl/cl_clientstartup.lua:29]: I don't like it, but we should decide when forming structs if we do
```lua
table.insert( cfcFactions.Alerts, {["Message"] = msg, ["Type"] = mtype, ["Time"] = os.date( "%T ", os.time() )} )
```
or
```lua
table.insert( cfcFactions.Alerts, {Message= msg, Type = mtype, Time = os.date( "%T ", os.time() )} )
```
(watch out for `Time` being a keyword, ya never know...)

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 3]
```diff
@@ -1,29 +1,29 @@
 surface.CreateFont( "CFC_Normal",
-    { 
+    {
         font = "arial",
         size = 18,
         weight = 500
     }
 )
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua:7]: Now these guys are what I'm talking about right here 👍

File: lua/cfcfactions/core/cl/dermas/cl_mainderma.lua [hunk 4]
```diff
@@ -81,9 +81,9 @@ end
 
 function Panel:Init()
     self:SetSize( math.Clamp( 1024, 0, ScrW() ), math.Clamp( 800, 0, ScrH() ) )
-    self:SetPos( ( ( ScrW() / 2 ) - ( self:GetWide() / 2 ) ), ( (ScrH() / 2 ) - ( self:GetTall() / 2 ) ) )
+    self:SetPos( ( ( ScrW() / 2 ) - ( self:GetWide() / 2 ) ), ( ( ScrH() / 2 ) - ( self:GetTall() / 2 ) ) )
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/cl/dermas/cl_mainderma.lua:84]: _muttering to self_
**not in this PR's scope not in this PR's scope not in this PR's scope not in this PR's scope not in this PR's scope**

File: lua/cfcfactions/core/sh/sh_playerext.lua [hunk 5]
```diff
@@ -1,4 +1,4 @@
---[[]
+-- [[]
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/sh/sh_playerext.lua]:
```suggestion

```

File: lua/cfcfactions/core/sh/sh_sharedutils.lua [hunk 6]
```diff
@@ -25,7 +25,7 @@ cfcFactions.Credits.Developers = {
     ["STEAM_0:1:13693373"] = "Lego1042"
 }
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/sh/sh_sharedutils.lua:26]: What a cool idea <3

File: lua/cfcfactions/core/sv/sv_dbhandler.lua [hunk 7]
```diff
@@ -64,15 +64,15 @@ function DB:LoadFactions()
...
 
---[[
+-- [[
     [Permissions DB Functions]
 ]]--
 
---[[
+-- [[
     [MISC DB Functions]
 ]]--
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/sv/sv_dbhandler.lua:73]: oh these are _block comments_...
Gross.

File: lua/cfcfactions/core/sv/sv_factionmanagment.lua [hunk 8]
```diff
@@ -9,75 +9,75 @@ local string = string
 local os = os
 local fpm = cfcFactions.fpm
 
---------------------------------------------------------------------------------------------------------------
---FACTIONS MANAGMENT COMMANDS
---------------------------------------------------------------------------------------------------------------
+-- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- ---
+-- FACTIONS MANAGMENT COMMANDS
+-- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- --- ---
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/sv/sv_factionmanagment.lua:14]: lmao

File: lua/cfcfactions/core/sv/sv_logging.lua [hunk 9]
```diff
@@ -18,18 +18,18 @@ cfcFactions.LogTypes = {
...
         Type = ERROR_TYPE,
         Args = {},
-        ShouldParse = true, --Parse args into the msg if possible
-        SaveDB = true, --Save to database?
+        ShouldParse = true, -- Parse args into the msg if possible
+        SaveDB = true, -- Save to database?
         Time = os.time(),
-        Checked = nil, --Allows us to check if this table has been sanitized before.
+        Checked = nil, -- Allows us to check if this table has been sanitized before.
     }
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/sv/sv_logging.lua:33]: YEAH BABY

File: lua/cfcfactions/core/sv/sv_mysql.lua [hunk 10]
```diff
@@ -68,12 +68,12 @@ function sql_db:initilize()
...
+                ) ENGINE = InnoDB DEFAULT CHARSET = latin1
             ]] ),
 
-        --Table to log all administrative actions IE: changing name, deleting faction, deleting user
+        -- Table to log all administrative actions IE: changing name, deleting faction, deleting user
         -- ( playerid:string, factionid:string, setting:string, state:string, time:string )
-        --State = {DELETED, MODIFIED, NEW}
+        -- State = {DELETED, MODIFIED, NEW}
 
         q3 = sql_db:query( [[
```
Comment (justinmuskopf) [inline lua/cfcfactions/core/sv/sv_mysql.lua:78]: I totally forgot about the Lua `[[` string thing

**Review by justinmuskopf — APPROVED** [review]
I made some comments about how we structure our `struct`-esque guys.
I'm going to argue that those are _technically_ whitespace changes and therefore in scope

Non-blocking IMO

File: lua/cfcfactions/core/cl/cl_clientstartup.lua (hunk 2 above)
Comment (sarahsturgeon) [reply lua/cfcfactions/core/cl/cl_clientstartup.lua:29]: MOONSCRIPT

**Comment by sarahsturgeon** [comment]
Closing in favor of: https://github.com/brandonsturgeon/cfc_factions_3/pull/55 (made with more advanced spacing changes)

**Pull request closed without merging**